guide/functions.md

```diff
@@ -25,13 +25,10 @@ today-is = Today is { DATETIME($date) }
 
 ## Function parameters
 
-Functions may (but don't have to) accept positional and keyword arguments. Some
-keyword parameters are only available to developers and cannot be used inside
-of FTL translations. Instead they can be set on external arguments passed into
-translations by the developer.
-
-See the reference below for more information about the arguments accepted for
-each built-in function.
+Functions may accept positional and keyword arguments. Some keyword parameters
```

Comment:
The docs are not up-to-date anymore here :-/

Reply:
Right, I missed this in #105. I'll fix it here since I'm already changing this line. Depending on the outcome of the discussion in #124, we might be able to fix this guide-wide there.